docs/contribute.rst

```diff
@@ -182,6 +182,17 @@ Here's the overview of core classes::
     │   └── GuestTestcloud
     └── Run
 
+Attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Object hierarchy is following: Run -> Plans -> Steps -> Plugin, where the Run is
+on the top of this hierarchy. The objects have the ``parent`` attribute, that is
+pointing on a parent from which the current object inherits from.
+
+``Node`` is a class which inherits from ``Common`` and points at the original fmf tree’s data.
```

Comment:
The class inheritance is covered above. Let's mention the `node` attribute instead, which contains reference to the original fmf node.